src/main/webapp/css/main.css

```diff
@@ -15,8 +15,12 @@
  */
 
 /* Prevent the page from becoming too wide. */
+.doc {
```

Comment:
Add a comment as to what this is doing.

Reply:
Ok, I made the change in the new pull request.